src/cli/services/linter/linter.ts

```diff
@@ -31,6 +31,8 @@ const KNOWN_FORMATS: Array<[string, FormatLookup, string]> = [
   ['json-schema-2019-09', isJSONSchemaDraft2019_09, 'JSON Schema Draft 2019-09 detected'],
 ];
 
+export const BUILTIN_FORMATS: Array<[string, FormatLookup]> = KNOWN_FORMATS.map(kf => [kf[0], kf[1]]);
```

Comment:
We should export it from a different file than CLI, ideally from the main entrypoint.
This looks like something one may move in future, so having it in main entrypoint will prevent any breakage.

Reply:
Yep I agree. We might need to make this stuff available for public usage but that doesn't mean it has to stay where it currently is.

Reply:
@nulltoken once you move this to a separate file, we should be good to go.